xbmc/input/KeyboardStat.cpp

```diff
@@ -32,6 +32,8 @@
 #include "threads/SystemClock.h"
 #include "utils/log.h"
 
+#define HOLD_THRESHOLD 250
```

Comment:
I assume you mean a configurable option?
We obviously can in the future, if there's a need